src/Controller/Plugin/Forward.php

```diff
@@ -193,7 +193,7 @@ protected function detachProblemListeners(SharedEvents $sharedEvents)
 
                     foreach ($classArray as $class) {
                         if ($currentCallback instanceof $class) {
-                            $sharedEvents->detach($id, $currentEvent);
+                            $sharedEvents->detach($currentEvent, $id);
```

Comment:
No. In the new version, the second argument is optional, the first required.
On Oct 1, 2015 6:25 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In src/Controller/Plugin/Forward.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40902934>:
>
> > @@ -193,7 +193,7 @@ protected function detachProblemListeners(SharedEvents $sharedEvents)
> >
> >                      foreach ($classArray as $class) {
> >                          if ($currentCallback instanceof $class) {
> > -                            $sharedEvents->detach($id, $currentEvent);
> > +                            $sharedEvents->detach($currentEvent, $id);
>
> This is a super-subtle bc break. Is it possible to revert this in the EVM
> instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40902934>.
>